### PR TITLE
Update recetox-xMSannotator to 0.10.0

### PR DESCRIPTION
**Pull request by xtrojak**
Update recetox-xMSannotator to 0.10.0. This introduced new dependencies and version pinning.

### CODE REVIEW

**Comment by xtrojak** [comment]
@BiocondaBot please add label

File: recipes/r-recetox-xmsannotator/meta.yaml [hunk 1]
```diff
@@ -8,42 +8,62 @@ package:
...
 
+channels:
+  - conda-forge
+  - bioconda
 
 requirements:
   host:
-    - r-base
-    - r-dplyr >=1
+    - r-base =4.1.1
```
Comment (bgruening) [inline recipes/r-recetox-xmsannotator/meta.yaml:27]: I would not pin it that strict if possible.

Comment (hechth) [reply recipes/r-recetox-xmsannotator/meta.yaml:27]: When using the environment script to create the environment, pinning the R version massively reduces the time needed to resolve. Not sure how this maps to the host section of a recipe.

**Comment by hechth** [comment]
Given the global pinnings for R, I think by being so specific we will make the package difficult to use in combination with any other package

**Review by hechth — APPROVED** [review]

**Pull request merged**